pyccel/decorators.py

```diff
@@ -130,6 +130,40 @@ def kernel(f):
     KernelAccessor
         A class representing the kernel function.
     """
+    class CudaThreadIndexing:
+        """
+        Class representing the CUDA thread indexing.
+
+        Class representing the CUDA thread indexing.
+        """
+        def __init__(self, block_idx, thread_idx):
+            self._block_idx = block_idx
+            self._thread_idx = thread_idx
+
+        def threadIdx(self, dim):
+            """
+            Get the thread index.
+
+            Get the thread index.
```

Comment:
Not sur why you repeat here? Either you don't want to follow the doc formats (which aren't enforced because this class is hidden inside a function) or you do and the parameter descriptions are missing

Reply:
The bot asks for documentation at https://github.com/pyccel/pyccel-cuda/pull/69/checks?check_run_id=27909189499, but I don't think it's necessary to include everything in the parameter and return sections

Reply:
Yes, pylint checks that there is documentation but nothing checks the format. So I would expect either:
```
            Get the thread index.
```
or
```
            Get the thread index.

            Get the thread index.

            Parameters
            -----------------
            dim : int
                   The dim argument that does something I guess
```

Reply:
You seem to have gone for something in the middle where you ignore the requirements that add useful information but keep the requirements that add useless information

Reply:
528099f is that better

Reply:
Yes great thanks